src/CWL/Encode.fs

```diff
@@ -1 +1,327 @@
 namespace ARCtrl.CWL
+
+open System
+open DynamicObj
+open YAMLicious
+open YAMLicious.YAMLiciousTypes
+open YAMLicious.Writer
+
+/// Encoding helpers and functions mirroring the logic in Decode.fs.
+/// Each decode path gets a corresponding encode path that produces
+/// YAML key ordering and scalar formatting matching existing fixtures.
+module Encode =
+    // ------------------------------
+    // Generic YAML node constructors
+    // ------------------------------
+    let private yScalar (s:string) =
+        YAMLElement.Value { Value = s; Comment = None }
+
+    let private yInt (i:int) = YAMLElement.Value { Value = string i; Comment = None }
```

Comment:
Shouldn't these generic functions be part of YAMLicious?

Reply:
done